### DISCUSSION TITLE
成为社区维护/志愿者！

### DISCUSSION THREAD

**Discussion by makerjackie** [obsidianzh/forum#3] [Ideas想法]
之前有社区成员陆续通过 开黑吧、知识星球、兔小巢、zulip 试图建立Obsidian中文社区，不过后面都失败了（活跃度不高，没有人使用），作为一枚程序员，常年使用 Github ，仅是出于兴趣，希望能够尽绵薄之力，**长期维护这个社区**。不过社区需要有维护者和积极解答问题的人（维护者），记得 star / watch 项目

如果有感兴趣的朋友，可以在这个帖子下留下你的github账号 + 加入ObsidianZh的原因，例如：
* 想要参与文档翻译
* 参与社区管理
* 翻译搬运工（从[英文论坛](http://forum.obsidian.md)中搬运一些实用的问答过来）

https://github.com/obsidianzh

**Comment by Quorafind** [comment]
本人: @Quorafind 
理由: 之前一直使用官方论坛以及社区经常讨论，受限于没有一个完全中文的社区，也十分遗憾，现在有机会参与到Github中文社区的建设，是个好机会

**Reply by makerjackie** [reply]
已发送邀请！

**Comment by ikelvinwoo** [comment]


**Reply by makerjackie** [reply]
欢迎！已发送邀请！关于翻译，有空的话可以fork[官方仓库](https://github.com/obsidianmd/obsidian-docs) 到我们组织obsidianzh下，我后期会参与，近期比较忙

**Comment by lillian-who** [comment]
我还以为这里是官方的社区，还提交了个bug，怪我没看清楚。
所以这里是大家互相交流使用经验的地方吗？

我想要成为社区自愿者。
同样是一枚程序员，多年笔记爱好者。嗯，不如说是效率工具爱好者，但非极客。
喜欢整理一些使用的分享，苦于生活中没有志同道合的人哈哈哈。
给你看看我的笔记：

![image](https://user-images.githubusercontent.com/20553685/102846174-4755b500-444a-11eb-84e7-247c137b1906.png)

**Reply by lillian-who** [reply]
普普通通使用小经验分享，社区需要我吗？

**Reply by cyddgh** [reply]
发布到其他地方，可以参考一下chrome的插件markdownhere

**Reply by makerjackie** [reply]
@YaRauy  欢迎！已发送邀请，欢迎你在这里和其他平台分享你的笔记心得，大家互相交流，共同进步！我也是效率工具爱好者

**Reply by lillian-who** [reply]
> 发布到其他地方，可以参考一下chrome的插件markdownhere

感谢！

**Comment by whitewatercn** [comment]
@whitewatercn
obsidian catalyst用户
经常上obsidian的论坛看消息，希望国内也有类似的，方便大家一起交流

翻译搬运工（从英文论坛中搬运一些实用的问答过来）
